extensions/objects/templates/math-expression-content-editor.directive.ts

```diff
@@ -35,15 +39,58 @@ angular.module('oppia').directive('mathExpressionContentEditor', [
       controllerAs: '$ctrl',
       controller: ['$scope', function($scope) {
         var ctrl = this;
+        var convertLatexStringToSvg = function(inputLatexString) {
+          var emptyDiv = document.createElement('div');
+          var outputElement = angular.element(emptyDiv);
+          var $script = angular.element(
+            '<script type="math/tex">'
+          ).html(inputLatexString === undefined ? '' : inputLatexString);
+          outputElement.html('');
+          outputElement.append($script);
+          MathJax.Hub.Queue(['Typeset', MathJax.Hub, outputElement[0]]);
+          MathJax.Hub.Queue(function() {
+            ctrl.svgString = (
+              outputElement[0].getElementsByTagName('svg')[0].outerHTML);
+          });
+        };
+
+        var processAndSaveSvg = function() {
```

Comment:
I don't see any saving functionality in this method is that expected?

Reply:
I've added a comment, yeah this is expected